internal/expr/query.go

```diff
@@ -12,35 +13,53 @@ func (qe *QueryExpr) QueryArgs() []any {
 	return qe.args
 }
 
+func (qe *QueryExpr) SQL() string {
+	return qe.sql
+}
+
 func (qe *QueryExpr) HasOutputs() bool {
 	return len(qe.outputs) > 0
 }
 
+// QueryExpr represents a complete SQLair query, ready for execution on a
+// database.
 type QueryExpr struct {
 	sql     string
 	args    []any
 	outputs []typeMember
 }
 
-// Query returns a query expression ready for execution, using the provided values to
+const markerPrefix = "_sqlair_"
```

Comment:
I would rather not change this function here because we are also moving it and it makes the changes hard to see. Relating to the comment above, if you wanted to bring a PR for that and change this there as well that would be great